internal/service/elb/listener_policy.go

```diff
@@ -38,6 +38,11 @@ func ResourceListenerPolicy() *schema.Resource {
 				Optional: true,
 				Elem:     &schema.Schema{Type: schema.TypeString},
 			},
+			"triggers": {
+				Type:     schema.TypeMap,
+				Optional: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+			},
```

Comment:
Is a `ForceNew` needed here?

Reply:
not needed here since this resource has and `resourceUpdate` handler. This will just trigger a "update" so the state refreshes